autogen/main/cluster.tf.tmpl

```diff
@@ -1054,7 +1054,7 @@ resource "google_container_node_pool" "windows_pools" {
         cpu_manager_policy                     = lookup(each.value, "cpu_manager_policy", "static")
         cpu_cfs_quota                          = lookup(each.value, "cpu_cfs_quota", null)
         cpu_cfs_quota_period                   = lookup(each.value, "cpu_cfs_quota_period", null)
-        insecure_kubelet_readonly_port_enabled = lookup(each.value, "insecure_kubelet_readonly_port_enabled", var.insecure_kubelet_readonly_port_enabled != null ? var.insecure_kubelet_readonly_port_enabled : null)
+        insecure_kubelet_readonly_port_enabled = lookup(each.value, "insecure_kubelet_readonly_port_enabled", null) != null ? upper(tostring(each.value.insecure_kubelet_readonly_port_enabled)) : null
```

Comment:
I don't think the second `lookup()` is needed here since we've already checked for null, but not sure if you think we should use dot or slice notation here (like `.foo` or `["foo"]`).

Reply:
Thanks @wyardley - I think using `each.value` directly is the best option, as you point out no need to guard against `null` as we just did that!  I'm fine with the dot notation.